public/assets/scss/player/_player.scss

```diff
@@ -174,6 +174,15 @@
 			margin: 0;
 			cursor: pointer;
 		}
+
+		.error-el{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space